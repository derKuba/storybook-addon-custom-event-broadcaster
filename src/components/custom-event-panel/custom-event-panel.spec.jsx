import React from "react";
import { shallow } from "enzyme";

import CustomEventPanel from "./custom-event-panel";

jest.mock("@storybook/api", () => ({
  __esModule: true,
  useParameter: () => null,
}));

describe("CustomEventPanel", () => {
  test("should render", () => {
    const wrapper = shallow(<CustomEventPanel />);
    expect(wrapper).toMatchSnapshot();
  });
});
