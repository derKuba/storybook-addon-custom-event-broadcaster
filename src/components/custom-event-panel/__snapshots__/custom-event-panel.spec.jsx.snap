// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CustomEventPanel should render 1`] = `
<div>
  <div>
    <thead>
      <tr>
        <th>
          EventName
        </th>
        <th>
          Data
        </th>
        <th>
          (opt.) Selector
        </th>
        <th>
           
        </th>
      </tr>
    </thead>
    <tbody>
      <tr>
        <td>
          <input
            className="css-12d6kl5"
            onChange={[Function]}
            type="text"
            value=""
          />
        </td>
        <td>
          <textarea
            className="css-12d6kl5"
            onChange={[Function]}
            value=""
          />
        </td>
        <td>
          <div
            className="css-k008qs"
          >
            <div
              className="css-11elljy"
            >
              <div />
            </div>
            <span>
              Add Selektor
            </span>
          </div>
        </td>
        <td>
          <button
            onClick={[Function]}
          >
            send
          </button>
        </td>
      </tr>
      <tr>
        <td>
          <div
            className="css-k008qs"
            onClick={[Function]}
          >
            <div
              className="css-11elljy"
            >
              <div />
            </div>
            <span>
              Add new Line
            </span>
          </div>
        </td>
      </tr>
    </tbody>
  </div>
</div>
`;
